packages/volto/locales/de/LC_MESSAGES/volto.po

```diff
@@ -2495,12 +2495,12 @@ msgstr "Objekt-Browser öffnen"
 #. Default: "Order"
 #: components/manage/Sidebar/Sidebar
 msgid "Order"
-msgstr ""
+msgstr "Reihung"
 
 #. Default: "Ordered"
 #: components/manage/Blocks/ToC/Schema
 msgid "Ordered"
-msgstr ""
+msgstr "Sortiert"
```

Comment:
This sets whether to use an `<ol>` element -- i.e. to produce a numbered list in the ToC. The list is not sorted in any way, but still shown in the order of the headings from the page's blocks.

Reply:
It is about using numbers vs dots? Then the term is  "Nummeriert".